src/bp-templates/bp-nouveau/buddypress/assets/emails/wp/email-template.php

```diff
@@ -43,7 +43,7 @@
 defined( 'ABSPATH' ) || exit;
 
 $settings = bp_email_get_appearance_settings();
-$width = wp_is_mobile() ? '100%' : '600px';
+$width = wp_is_mobile() ? '100%%' : '600px';
```

Comment:
@meetmakadiya Can you run PHPCS formating on this?

Reply:
@dipskakadiya yes I run the PHPCS and have not found any kind of issue.
[report.txt](https://github.com/buddyboss/buddyboss-platform/files/7576754/report.txt)